terraform/ec2-suse-builders/main.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  suse_matrix_path = "../../versions/slesMatrix.json"
```

Comment:
This file is generated by the `generateMatrices` Make target, prior to executing the `terraform apply/destroy`.